osmosis/assetlist.json

```diff
@@ -1117,6 +1117,55 @@
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/persistence/images/stkosmo.png",
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/persistence/images/stkosmo.svg"
       }
+    },
+    {
+      "description": "The PAGE token is used for actions in the PageDAO NFT literary ecosystem and for DAO governance.",
+      "denom_units": [
+        {
+          "denom": "ibc/23A62409E4AD8133116C249B1FA38EED30E500A115D7B153109462CD82C1CD99",
+          "exponent": 8,
+          "aliases": [
+            "PAGE"
+          ]
+        },
+        {
+          "denom": "PAGE.grv",
+          "exponent": 8
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/23A62409E4AD8133116C249B1FA38EED30E500A115D7B153109462CD82C1CD99",
+      "name": "PageDAO",
+      "display": "Page",
```

Comment:
Please correct either this value 'Page' -> 'PAGE.grv' to match the denom_unit, or to follow the conversion of the others, you could correct the denom_unit and this value to just 'page'.